src/ui/DynamicFacet/DynamicFacetValues/DynamicFacetRangeValueCreator.ts

```diff
@@ -44,7 +44,11 @@ export class DynamicFacetRangeValueCreator implements ValueCreator {
 
     return new DynamicFacetValue(
       {
-        ...responseValue,
+        state: responseValue.state,
+        numberOfResults: responseValue.numberOfResults,
+        start: responseValue.start,
+        end: responseValue.end,
+        endInclusive: responseValue.endInclusive,
         value,
```

Comment:
Should we have a test checking that the CategoryFacet request has the correct values?

Reply:
This is the Dynamic Facet, since I updated the response the spread wasn't working anymore